docs/BenchmarkTiDB.md

```diff
@@ -0,0 +1,109 @@
+# Benchmark TiDB
```

Comment:
It's better to rename it as `benchmark-tidb.md`

Reply:
done